LondonBritishLibrary/orient/BLorient13266.xml

```diff
@@ -0,0 +1,419 @@
+                                        <desc type="Comment">Short marginal notes written in another hand.</desc>
+                                    </item>
+                                    <item xml:id="e1">
+                                        <locus target="#180v"/>
+                                        <desc type="OwnershipNote">The owner of <ref target="p1"/> was 
+                                            <persName role="owner" ref="PRS14366HabtaMaryam"/>.</desc>
+                                    </item>
+                                    <item xml:id="e2">
+                                        <locus target="#36r #37v #38r #44r #48v #91v #102r #123v"/>
+                                        <desc type="Unclear">Scribbles, pen trials.</desc>
+                                    </item>
+                                    <item xml:id="e3">
+                                        <locus target="#63v"/>
+                                        <desc type="Correction">After the end of Psalm 65 two lines are erased.</desc>
+                                    </item>
+                                </list>
+                            </additions>  
+                        </physDesc>
+                        <history>
+                            <origin>
+                                <origDate notBefore="1700" notAfter="1800">18th century</origDate>
+                            </origin>
+                        </history>
+                    </msPart>
+                    <msPart xml:id="p2">
+                        <msIdentifier>
+                            <idno/>
+                        </msIdentifier>
+                        <msContents>
+                            <msItem xml:id="p2_i1">
+                                <locus from="194ra" to="201vb"/>
+                                <title ref="LIT1356eqaban"/>
+                                <incipit xml:lang="gez">ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ ዕቀበኒ፡ በእንተ፡ ዘኀደርከ፡ በከርሠ፡ ማርያም። ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ ዕቀበኒ፡ በእንተ፡ ማርያም፡
+                                    አፍቂረከ፡ ዘኀደርከ፡ በውስተ፡ ማኅፀና፡ ለማርያም፡ ድንግል። ዕቀበኒ፡ <corr resp="PRS8999Strelcyn">ኦ</corr>እግዚእየ፡ በእንተ፡ ፍቅሩ፡ ወበእንተ፡
+                                    ዘሐቀፍኪዮ፡ በአእዳውኪ፡ ወበእንተ፡ ዘአጥበውኪዮ፡ ሐሊበ፡ ውእቱ፡ እግዚእነ፡ እየሱስ፡ ክርስቶስ። ዕቀበኒ፡ በእንተ፡ ፍቅራ፡ ለማርያም፡ እግዚአየ፡ ኢየሱስ፡ ክርስቶስ፡ 
+                                        </incipit>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf">6</measure>
+                                        <dimensions type="outer" unit="mm">
```

Comment:
could you repeat  `<locus from="194ra" to="201vb"/>` here to indicate the position of this unit within the codex? (it would also be good to do the same for p1)